meerkat_nest/util/__init__.py

```diff
@@ -2,16 +2,26 @@
 Meerkat Nest utility functions
 """
 from meerkat_nest import config
+import hashlib
 
 
 def scramble(value):
     """
     scrambles the input string value
+
+    Returns:\n
+        Returns an empty string
     """
     ret = ''
     return ret
 
 
+def hash(value):
+    encrypted_value = hashlib.sha256(value.encode('utf-8') + config.salt).hexdigest()
```

Comment:
hashlib requires byte stream as input. What type is `config.salt`?
Also the tmp variable should say `hashed_value`. It's not encrypted.
You can do go for simple and direct return here:
`return hashlib.sha256(...)`